tests/paddle/test_layers.py

```diff
@@ -1328,7 +1302,7 @@ def calc_transformer_output_and_grad(layer,
         if output_layernorm:
             assert_allclose(layer_te.self_attention.qkv.bias.grad,
                             layer_pd.self_attention.qkv.bias.grad,
-                            rtol=0.01,
+                            rtol=0.5,
```

Comment:
This is quite a loose tolerance. Would it help to make the tensor dims much smaller, something like `seq_len=32`, `num_heads=2`, `hidden_size=64`, so that we're not getting much numerical error from the GEMMs? The attention kernels should be tested in cuDNN, so we just want to check that Paddle is calling them correctly.